src/messages/MMgrConfigure.h

```diff
@@ -16,14 +16,15 @@
 #define CEPH_MMGRCONFIGURE_H_
 
 #include "msg/Message.h"
+#include "mgr/OSDPerfMetricQuery.h"
 
 /**
  * This message is sent from ceph-mgr to MgrClient, instructing it
  * it about what data to send back to ceph-mgr at what frequency.
  */
 class MMgrConfigure : public Message
 {
-  static const int HEAD_VERSION = 2;
+  static const int HEAD_VERSION = 4;
```

Comment:
When you add multiple fields in a single PR, you can just increment the version by 1, no need to increment it more than once per-attribute.

Reply:
... and only once per release (i.e. this structure can keep changing now in the master branch w/o incrementing the version since we don't maintain backwards compatibility w/ dev releases).

Reply:
Ok, thanks for the info - i'll have it fixed